Tests/SwinjectStoryboard/Storyboard+SwizzlingSpec.swift

```diff
@@ -1,21 +1,30 @@
 //
-//  UIStoryboard+SwizzlingSpec.swift
+//  Storyboard+SwizzlingSpec.swift
 //  Swinject
 //
 //  Created by Yoichi Tagaya on 10/10/15.
 //  Copyright © 2015 Swinject Contributors. All rights reserved.
 //
 
+
 import Quick
 import Nimble
 @testable import Swinject
+    
+#if os(iOS) || os(tvOS)
+private typealias Storyboard = UIStoryboard
+#elseif os(OSX)
+private typealias Storyboard = NSStoryboard
+#endif
+
+#if os(iOS) || os(OSX) || os(tvOS)
 
-class UIStoryboard_SwizzlingSpec: QuickSpec {
+class Storyboard_SwizzlingSpec: QuickSpec {
```

Comment:
Type Name Violation: Type name should only contain alphanumeric characters: 'Storyboard_SwizzlingSpec' (type_name)